ext/pycall/pycall.c

```diff
@@ -1201,6 +1209,7 @@ pycall_libpython_helpers_m_delitem(VALUE mod, VALUE pyptr, VALUE key, VALUE v)
   if (res == -1) {
     pycall_pyerror_fetch_and_raise("PyObject_DelItem");
   }
+  pycall_Py_DecRef(pyobj_key);
```

Comment:
I think this `DecRef` should be called just after calling `PyObject_DelItem`.

Reply:
OK